packages/action-group/src/ActionGroup.ts

```diff
@@ -317,7 +318,8 @@ export class ActionGroup extends SpectrumElement {
         }
         if (
             (changes.has('quiet') && this.quiet) ||
-            (changes.has('emphasized') && this.emphasized)
+            (changes.has('emphasized') && this.emphasized) ||
+            (changes.has('size') && this.size)
```

Comment:
Has a cascade effect on the Action Button stories that had previously leveraged Action Group without a size, and not the default to `m` causes visual effects. This is a nice feature add, but it'll be important to include this information in our release announcement.